<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CtrlDoublePubStuWdg</class>
 <widget class="QWidget" name="CtrlDoublePubStuWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>49</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#CtrlDoublePubStuWdg
{
background-color: rgb(61, 64, 70);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="cmbUserList">
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>38</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>131</width>
       <height>38</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
font: 14px &quot;微软雅黑&quot;;
    border-radius: 3px;
    padding: 0px 1px 1px 5px;
    background-color: rgb(49, 52, 57);
    color: rgb(236, 236, 236);
padding-left:19px;

}

QComboBox::drop-down{
width: 15px;
border-left: 0px; 
}  

QComboBox::down-arrow 
{
image: url(:/doublePub/img/doublePub/arrow.png);
height:38px;width:25px;
margin:0px,0px,0px,0px;
padding-right:19px;
}

QComboBox QAbstractItemView{
    outline: 0px;
    border-radius:4px;
	color: rgb(236, 236, 236);
	selection-background-color: rgb(49, 52, 57);
    selection-border-radius:4px;
	selection-color: rgb(82, 204, 144);
	background-color: rgb(61, 64, 70);
	border-left:15px;
	 border: 1px solid rgb(61, 64, 70);
}

/*滚动条样式 开始*/
QScrollBar:vertical
{
border-radius:4px;
width:6px;
background:rgba(255,255,255, 255);
margin:0px,0px,0px,0px;

}

QScrollBar::handle:vertical
{
background:rgba(210,210,210, 255);
border-radius:3px;
min-height:20;
border: 1px solid #D2D2D2;
}    </string>
     </property>
     <item>
      <property name="text">
       <string>新建项目</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>新建项目</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnPre">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>11</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>11</width>
       <height>16</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-image: url(:/doublePub/img/doublePub/LeftRight.png)  0 11 32 0;
}
                                                    
QPushButton:hover
{
border-image: url(:/doublePub/img/doublePub/LeftRight.png) 16 11 16 0;
}

QPushButton:pressed
{
border-image: url(:/doublePub/img/doublePub/LeftRight.png) 0 11 32 0;
}

QPushButton:disabled
{
border-image: url(:/doublePub/img/doublePub/LeftRight.png) 32 11 0 0;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnNext">
     <property name="minimumSize">
      <size>
       <width>11</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>11</width>
       <height>16</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-image: url(:/doublePub/img/doublePub/LeftRight.png) 0 0 32 11;
}
                                                    
QPushButton:hover
{
border-image: url(:/doublePub/img/doublePub/LeftRight.png) 16 0 16 11;
}

QPushButton:pressed
{
border-image: url(:/doublePub/img/doublePub/LeftRight.png) 0 0 32 11;
}

QPushButton:disabled
{
border-image: url(:/doublePub/img/doublePub/LeftRight.png) 32 0 0 11;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="GifButton" name="btnClose">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>14</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>14</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-image: url(:/doublePub/img/doublePub/close.png) 0 14 0 0;
}
                                                    
QPushButton:hover
{
border-image: url(:/doublePub/img/doublePub/close.png) 0 0 0 14;
}

QPushButton:pressed
{
border-image: url(:/doublePub/img/doublePub/close.png)0 14 0 0;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GifButton</class>
   <extends>QPushButton</extends>
   <header>gifbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../VLive/VLive.qrc"/>
 </resources>
 <connections/>
</ui>
